MAINTAINERS.md

```diff
@@ -1,16 +1 @@
-The Linkerd2 Proxy maintainers are:
-
-* Oliver Gould <ver@buoyant.io> @olix0r (super-maintainer)
-* Eliza Weisman <eliza@buoyant.io> @hawkw
-* Sean McArthur <sean@buoaynt.io> @seanmonstar
-* Alex Leong <alex@buoyant.io> @adleong
-
-<!--
-# Adding a new maintainer
-
-* Submit a PR modifying this file
-* Add maintainer to .github/CODEOWNERS
-* Obtain approvals per GOVERNANCE.md
-* Invite maintainer to https://github.com/orgs/linkerd/teams/linkerd2-maintainers/members
-* Invite maintainer to https://github.com/orgs/linkerd/people
--->
+See https://github.com/linkerd/linkerd2/blob/main/MAINTAINERS.md
```

Comment:
Take it or leave it: might be worth making this a Markdown link so people can just click it?